widgets/src/other/Legend.js

```diff
@@ -0,0 +1,106 @@
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["../common/SVGWidget", "../chart/I2DChart"], factory);
+    } else {
+        root.Entity = factory(root.SVGWidget);
+    }
+}(this, function (SVGWidget,I2DChart) {
+    function Legend(target) {  
```

Comment:
There should be no arguments (target) here - I just did a search through all the sources and it seems this is in a lot of places - I will tidy that up as a separate PR (amazing how you don't see something like this when you look at it every day!)...